test/test_transforms.py

```diff
@@ -398,6 +408,25 @@ def test_pad_with_mode_F_images(self):
         padded_img = transform(img)
         self.assertSequenceEqual(padded_img.size, [edge_size + 2 * pad for edge_size in img.size])
 
+    def test_pad_negative(self):
```

Comment:
Can you put and adapt this test into https://github.com/pytorch/vision/blob/b5664ea82405c37b1f394bf4f70d6ecef3a35b11/test/test_transforms_tensor.py#L123

Reply:
@jamt9000 it is not the same `test_pad` :) It is one inside `test_transforms_tensor.py`. Thanks !

Reply:
Oops, changed the right one now